freqtrade/freqai/data_drawer.py

```diff
@@ -176,24 +179,34 @@ def load_historic_predictions_from_disk(self):
             except EOFError:
                 logger.warning(
                     'Historical prediction file was corrupted. Trying to load backup file.')
-                with self.historic_predictions_bkp_path.open("rb") as fp:
-                    self.historic_predictions = cloudpickle.load(fp)
+                self.historic_predictions = pd.read_parquet(
+                    self.historic_predictions_bkp_path_parquet)
                 logger.warning('FreqAI successfully loaded the backup historical predictions file.')
 
+        elif not exists and convert:
+            logger.info("Converting your historic predictions pkl to parquet"
+                        "to improve performance.")
+            with Path.open(self.historic_predictions_path, "rb") as fp:
+                self.historic_predictions = cloudpickle.load(fp)
+            self.historic_predictions.to_parquet(self.historic_predictions_path_parquet)
```

Comment:
what made you pick parquet over feather?

based on my [tests](https://www.freqtrade.io/en/latest/data-download/#dataformat-comparison) - feather is both slightly smaller, as well as slightly faster ... (and the interface to it is effectively identical, except for a different function name (read_feather -> read_parquet, ...).

Reply:
I read that Feather is not good for long-term storage. If you think they have alleviated that issue, then I would be perfectly happy to switch to feather.